src/Configuration/Config.Binder/test/ConfigurationBinderTests.cs

```diff
@@ -387,13 +387,13 @@ public void ConsistentExceptionOnFailedBinding(Type type)
             Assert.NotNull(exception.InnerException);
             Assert.NotNull(getException.InnerException);
             Assert.Equal(
-                Resources.FormatError_FailedBinding(IncorrectValue, type),
+                Resources.FormatError_FailedBinding(IncorrectValue, "Value", type),
```

Comment:
Can you add a test that demonstrates what this change actually would look like with a path?

Reply:
Doesn't it already demonstrate it with this test?
Using an in memory collection as the configuration provider, it's kind of forced to be a flat key/value map, so adding `A:More:Elaborate:Path` doesn't really prove anything. It's basically the same thing.
But if you disagree, I can add it.

Reply:
Or are you looking for an example of binding to a deeper object graph?

Reply:
Yes, a deeper path test that demonstrates how this would be useful

Reply:
Test added